arrows/klv/klv_0601.cxx

```diff
@@ -1283,9 +1283,7 @@ operator<<( std::ostream& os, klv_0601_control_command const& value )
 {
   return os << "{ " << "ID: " << value.id
             << ", String: \"" << value.string
-            << "\", Timestamp: " << ( value.timestamp
-                                  ? std::to_string( value.timestamp )
-                                  : "(empty)" )
+            << "\", Timestamp: " << value.timestamp
```

Comment:
I think I remember that the optional type takes care of printing the empty value. In that case, there are some 0601 prints that should be updated. (LGTM)